### DISCUSSION TITLE
Trouble connecting to Wyze Cam V3

### DISCUSSION THREAD

**Discussion by MatthewTingum** [HclX/WyzeHacks#153] [General]
I bring this discussion here for a few reasons:
- I have modified the firmware to include a new root user with known credentials
  - This may have caused my issue
- I have a UART connection active
  - That again, may be causing this issue
- We may be able to diagnose this issue further with a root busybox shell
  - I have a more fully featured busybox than the one running on the camera

## Background

I have a Wyze Cam V3 (Not Pro) running firmware version 4.36.10.4054.

I wanted code execution so I started by chip clipping the SPI flash. No luck detecting the flash, so I lifted it and read the SPI out of circuit. I then added a user with creds `user:password`. After replacing the SPI flash I was successful logging in over UART with the creds `user:password`. At the time, I was able to upload my own copy of busybox, launch a telnet daemon, and go wild.

I put together a similar setup for a friend so we could poke around the Wyze together. The only problem was that the 2nd camera I modified would not connect to my wifi. I was left thinking that I might have damaged something in the process.

A day passes and I'm no longer able to connect to wifi on the first camera I had modified. Weird... I hadn't really moved the camera and the solder joints to the UART and SPI flash are pretty steady. This pushed me to finally replace the router I'm renting from an ISP with my own router. After doing so it worked again! I just had to hook up UART again and launch the telnet daemon. I hooked up UART on the live system but got no response. No worries, I'll just restart the camera. After restarting the camera, I was able to interact with the UART shell. But this time, it refused to connect to the wifi again.

## How can this be fixed?

Maybe I try to change my SSID again. I don't know exactly what fixed it between router swaps. One thing I do know is that the SSID changed. It has successfully connected to the first and second router at one point.

???

I'm open to suggestions. Curious to know if others have run into this.

**Comment by virmaior** [comment]
Not sure on the fixing but the my understanding is that WyzeHacks is not being actively maintained at this point.
In contrast, https://github.com/gtxaspec/wz_mini_hacks is still making strides in doing cool things with Wyze V2 and V3 cameras.
for V3s the cameras check FAT32 cards for a particular file which can then execute arbitrary code.
For V2s the firmware has to be recompiled.

**Reply by MatthewTingum** [reply]
Ah, I've seen that the cam checks for `/media/mmc/test.tar` or something to that extent on boot on the Wyze cam v3. Maybe that's what you're referencing. I'll direct my question that way. Thanks for the quick response!
